library/gcm.c

```diff
@@ -278,7 +278,7 @@ int mbedtls_gcm_starts( mbedtls_gcm_context *ctx,
     else
     {
         memset( work_buf, 0x00, 16 );
-        MBEDTLS_PUT_UINT32_BE( iv_len * 8, work_buf, 12 );
+        MBEDTLS_PUT_UINT64_BE( iv_len * 8,  work_buf, 8 );
```

Comment:
Nit: extra space before `work_buf`
```suggestion
        MBEDTLS_PUT_UINT64_BE( iv_len * 8, work_buf, 8 );
```

Reply:
thanks